component-generated/src/app.rs

```diff
@@ -110,6 +110,10 @@ impl App {
         let Some(keymap) = self.config.keybindings.get(&self.mode) else {
             return Ok(());
         };
+        if self.components.iter().any(|c| c.is_editing()) {
+            action_tx.send(Action::RawKeyEvent(key))?;
+            return Ok(());
+        }
```

Comment:
If hypothetically someone wanted to add a key binding for `Action::ExitEditingMode` to their config.toml, with this they wouldn't be able to get out of the editing mode, right? Because once they enter editing this will always send raw key events.